kpops/pipeline_generator/pipeline.py

```diff
@@ -304,14 +313,14 @@ def validate(self) -> None:
         self.components.validate_unique_names()
 
     @staticmethod
-    def pipeline_filename_environment(path: Path, config: KpopsConfig) -> Path:
+    def pipeline_filename_environment(path: Path, environment: str) -> Path:
```

Comment:
`path` is maybe not the best name anymore. Let's update it. How about `pipeline_path`?

Reply:
Agreed